examples/webgl_clipculldistance.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>three.js webgl2 - clip cull distance</title>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0" />
+    <link
+      type="text/css"
+      rel="stylesheet"
+      href="main.css" />
+  </head>
+  <body>
+    <div id="container"></div>
+
+    <div id="info">
+      <a
+        href="https://threejs.org"
+        target="_blank"
+        rel="noopener"
+        >three.js</a
+      >
+      - clip cull distance
+      <a
+        href="https://registry.khronos.org/webgl/extensions/EXT_clip_cull_distance/"
+        target="_blank"
+        rel="noopener"
+        >EXT_clip_cull_distance</a
```

Comment:
Please use the `WEBGL_` extension name and link.

Reply:
Fixed!